INSTALL.md

```diff
@@ -80,8 +80,9 @@ See [the rustc-dev-guide for more info][sysllvm].
    ```
 
    If you plan to use `x.py install` to create an installation, it is
-   recommended that you set the `prefix` value in the `[install]` section to a
-   directory: `./configure --set install.prefix=<path>`
+   recommended that you set the `prefix` and `sysconfdir` values in the
+   `[install]` section to a directory: `./configure --set install.prefix=<path>
+   --set install.sysconfdir=<path>`
```

Comment:
Would be great to include [`DESTDIR`](https://rustc-dev-guide.rust-lang.org/building/new-target.html?highlight=DESTDIR#cross-compiling) here.

Reply:
Sure.